### DISCUSSION TITLE
Problem with a tray icon

### DISCUSSION THREAD

**Discussion by shasoka** [hiddify/hiddify-app#961] [Q&A]
> I'm using Hiddfy Client 1.4.0 on Windows 11 23H2

Situation description:
    1. I restart or simply start my PC
    2. Logging in my User Account
    3. Waiting for all the autoruns done
    4. Expecting to see the Hiddify tray icon (picture 1)
    5. Variant a: Not seeing it at all, while the Hiddify can be found in Windows Task Manager; and Variant b: Tray icon is hidden in taskbar overflow section, while it has been moved to the taskbar's main section at previous launch (and, ofcourse, there is Variant c: Tray icon is on it's place)
    6. Then for a I relaunch Hiddify via Task Manager and for b I drag and drop the icon from overflow to main section
![image](https://github.com/hiddify/hiddify-next/assets/90062361/bde919ca-354b-4590-98b3-16896ccbb187)
Picture 1 - My expectations...

And what's most strange is that in Windows Settings I see many instances of this icon (picture 2), which I try to turn on or off (hoping for the best)
![ApplicationFrameHost_06_10_24_17_08](https://github.com/hiddify/hiddify-next/assets/90062361/77e4ef6f-7208-4673-900c-3f5e2d8924f6)
Picture 2 - Too many instances!

Never faced such problem with other apps, hope it can be fixed.

Thank you for your hard work, comrade. Really like your product. B-)

**Comment by banta84** [comment]
Bug still exists!

**Comment by karinayakimova** [comment]
Hello! I have nearly the same issue. Then Hiddify changes status the icon dissapears from system tray. I need to manual drop it in the system tray for visibility. Win 10 Pro 1909. Is there any fixes?
